### PR TITLE
BF: scil_visualise_gradients

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by scil-admin** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-482/1/

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Review by mdesco — APPROVED** [review]
@GuillaumeTh please merge if you can.

File: scripts/scil_visualize_gradients.py [hunk 1]
```diff
@@ -96,36 +99,47 @@ def main():
...
-        gradient_sampling_file = args.gradient_sampling_file[0]
-        tmp = np.genfromtxt(gradient_sampling_file, delimiter=' ')
+        in_gradient_scheme = args.in_gradient_scheme[0]
+        tmp = np.genfromtxt(in_gradient_scheme, delimiter=' ')
         points = tmp[:, :3]
         bvals = tmp[:, 3]
         centroids, shell_idx = identify_shells(bvals)
 
+    if args.verbose:
+        print("Found {} centroids: {}".format(len(centroids), centroids))
```
Comment (GuillaumeTh) [inline scripts/scil_visualize_gradients.py:120]: Please use logging instead of print

File: scripts/scil_visualize_gradients.py [hunk 2]
```diff
@@ -96,36 +99,47 @@ def main():
...
         possible_output_paths = [out_basename + '_shell_' + str(i) +
                                  '.png' for i in centroids]
         possible_output_paths.append(out_basename + '.png')
         assert_outputs_exist(parser, args, possible_output_paths)
 
-    for idx, b0 in enumerate(centroids[centroids < 40]):
+    indexes = []
+    for idx in np.where(centroids < 40)[0]:
+        if args.verbose:
+            print("Removing bval = {} from display".format(centroids[idx]))
```
Comment (GuillaumeTh) [inline scripts/scil_visualize_gradients.py:132]: Please use logging instead of print

**Review by GuillaumeTh — APPROVED** [review]
Two small comments. Can be merged after.

**Pull request merged**